linker/Linker/Driver.cs

```diff
@@ -340,6 +346,7 @@ static void Usage (string msg)
 			Console.WriteLine ("   --dump-dependencies Dump dependencies for the linker analyzer tool");
 			Console.WriteLine ("   --reduced-tracing   Reduces dependency output related to assemblies that will not be modified");
 			Console.WriteLine ("   --used-attrs-only   Attributes on types, methods, etc will be removed if the attribute type is not used");
+			Console.WriteLine ("   --no-security-attrs Attributes on assemblies, types, and methods related to security will be removed");
```

Comment:
My preference would be `--strip-security` and you should note somewhere it works only when assembly is linked